### DISCUSSION TITLE
I am not able to access my profile page

### DISCUSSION THREAD

**Discussion by SaxenaAnushka102** [github/india#847] [Students]
I am not able to access my profile page due to the error:
"An unhandled exception occurred while processing the request.
JsonReaderException: Unexpected character encountered while parsing value: <. Path '', line 0, position 0.
Newtonsoft.Json.JsonTextReader.ParseValue()"

Also, I am not able to submit my AZ-900 certification on the Future Ready Skills portal due to same error. Please suggest some solution!

**Comment by ey-mentor1** [comment]
Hello,
Please check now
Thank you

**Comment by ey-mentor1** [comment]
If issue still comes.
Please drop mail to FRT support team.also share mail details here too , I'll forward it too.
Thank you

**Reply by SaxenaAnushka102** [reply]
It still isn't working. I have already written to the support regarding the same but haven't received any response yet.
Please suggest some solution ASAP.
email: anushka.saxena.iips@gmail.com

**Reply by ey-mentor1** [reply]
Hello,
Can you please share the detailed issue and email screenshot here. I'll send mail to them.
Please add your account ID also in that mail.
Thank you

**Reply by SaxenaAnushka102** [reply]
![Screenshot (27)](https://user-images.githubusercontent.com/58849657/169658874-52cd345c-3a3a-4f72-bf23-b3653be3ab16.png)
I AM NOT ABLE TO ACCESS MY PROFILE PAGE and I am not able to upload my AZ-900 certification on the Future Ready Talent portal. The page displays an internal server error as:
"An unhandled exception occurred while processing the request.
JsonReaderException: Unexpected character encountered while parsing value: <. Path '', line 0, position 0."

Please suggest some solution to the issue as soon as possible.
Please find the screenshot of the error in the attached file.
email: anushka.saxena.iips@gmail.com

**Reply by SaxenaAnushka102** [reply]
![Screenshot (26)](https://user-images.githubusercontent.com/58849657/169659021-af55a5b1-2e99-4056-b396-54df7bb3d25d.png)

**Reply by SaxenaAnushka102** [reply]
> If issue still comes. Please drop mail to FRT support team.also share mail details here too , I'll forward it too. Thank you

Hello! The issue still persists. Please notify if there are any updates?

**Comment by Srikarpavanbaba** [comment]
Unable to access my account and it's still not working my submission date also expired but iam not able to open the future ready talent 
![Screenshot_20221211-223200_Chrome](https://user-images.githubusercontent.com/116246231/206917662-ceacda06-833a-4302-b711-f9f298c8da56.jpg)
